include/deal.II/algorithms/any_data.h

```diff
@@ -76,7 +76,7 @@ class AnyData : public Subscriptor
    * use try_read() instead.
    */
   template <typename type>
-  const type
+  type
   entry(const std::string &name) const;
```

Comment:
Could `type` be a reference/pointer? And if so, would that make the pointer const, or the pointed to object const?

Reply:
The `const` doesn't matter in that case either, see https://godbolt.org/z/8jesMca3a. Returning a constant pointer is the same as returning a non-constant pointer and a pointer to a constant type is still returned as such.